battleship-client/src/main/java/com/epam/solid/nie/client/communication/SocketClient.java

```diff
@@ -40,7 +41,13 @@ public void send(String allShips) throws IOException {
         enemyShips = in.readLine();
     }
 
+    @Override
     public String getEnemyShips() {
         return enemyShips;
     }
+
+    @Override
+    public void sendGameOverToOpponent() {
```

Comment:
javadoc

Reply:
there is separate task for adding javadocs

Reply:
@Ozymadias could you link it please?

Reply:
https://github.com/smolamarcin/battleship/issues/151